src/connectedvmware/azext_connectedvmware/_help.py

```diff
@@ -31,8 +31,9 @@
       - name: Create resource pool
         text: |-
                az connectedvmware resource-pool create --custom-location "custom location name" \
-                --location "" --mo-ref-id "mo-ref id of the resource in vc" --name "resource pool name" \
-                --resource-group "resource group name" --vcenter "name or id of the vcenter"
+               --location "location name" --inventory-item "name or id the inventory item" \
+               --name "resource pool name" --resource-group "resource group name" --vcenter \
+               "name or id of the vcenter"
```

Comment:
It is recommended to change the entire parameter to a new line instead of changing to a new line in the middle of the parameter value, which will be more readable
```suggestion
               --name "resource pool name" --resource-group "resource group name" \
               --vcenter "name or id of the vcenter"
```